runtime/compiler/optimizer/VectorAPIExpansion.cpp

```diff
@@ -1212,7 +1258,10 @@ TR::Node *TR_VectorAPIExpansion::transformLoadFromArray(TR_VectorAPIExpansion *o
    TR::Node *aladdNode = generateAddressNode(array, arrayIndex, elementSize);
```

Comment:
I believe `elementSize` must be 1 for mask load. Otherwise, the offset into the byte array is incorrectly scaled as in the ILtree below.
```
 198804 n9134n      i2mVector128Int32 ()                                                              [       0x10c4616d0] bci=[30,47,208] rc=2 vc=5787 vn=- li=- udi=- nc 198804 =1 flg=0x28
 198805 n10244n       iloadi  <array-shadow>[#206  Shadow] [flags 0x80000603 0x0 ]                    [       0x10c7671c0] bci=[30,47,208] rc=1 vc=0 vn=- li=- udi=- nc=1
 198806 n10233n         aladd                                                                         [       0x10c766e50] bci=[-1,0,40] rc=1 vc=0 vn=- li=- udi=- nc=2
 198807 n9110n            ==>aload
 198808 n10231n           ladd                                                                        [       0x10c766db0] bci=[-1,0,40] rc=1 vc=0 vn=- li=- udi=- nc=2
 198809 n10229n             lshl                                                                      [       0x10c766d10] bci=[-1,0,40] rc=1 vc=0 vn=- li=- udi=- nc=2
 198810 n9121n                ==>lload
 198811 n10230n               iconst 2                                                                [       0x10c766d60] bci=[-1,0,40] rc=1 vc=0 vn=- li=- udi=- nc=0
 198812 n10232n             lconst 24                                                                 [       0x10c766e00] bci=[-1,0,40] rc=1 vc=0 vn=- li=- udi=- nc=0
```